devops/scripts/setup-playwright-tests.sh

```diff
@@ -0,0 +1,187 @@
+  echo ""
+  # Single site.
+  if [ "${type}" == 'single' ]; then
+    echo -e "${YELLOW}Install (Single Site) WordPress${RESET}"
+    terminus wp "${site_id}".dev -- core install --title="${site_name}" --admin_user=wpcm --admin_email=test@dev.null
+  fi
+
+  # Subdirectory multisite.
+  if [ "${type}" == 'subdir' ]; then
+    echo -e "${YELLOW}Install (Subdirectory Multisite) WordPress${RESET}"
+    terminus wp "${site_id}".dev -- core multisite-install --title="${site_name}" --admin_user=wpcm --admin_email=test@dev.null --subdomains=false --url="${site_url}"
+  fi
+
+  # Subdomain multisite.
+  if [ "${type}" == 'subdom' ]; then
+    echo -e "${YELLOW}Install (Subdomain Multisite) WordPress${RESET}"
+    terminus wp "${site_id}".dev -- core multisite-install --title="${site_name}" --admin_user=wpcm --admin_email=test@dev.null --subdomains=true --url="${site_url}"
+  fi
+
+  terminus wp "${site_id}".dev -- option update permalink_structure '/%postname%/'
+  terminus wp "${site_id}".dev -- rewrite flush
+  terminus wp "${site_id}".dev -- cache flush
+}
+
+status_check() {
+  echo ""
+  echo -e "${YELLOW}Checking WordPress install status${RESET}"
+  terminus wp "${site_id}".dev -- cli info
+  if [ "${type}" != 'single' ]; then
+    if ! terminus wp "${site_id}".dev -- config get MULTISITE; then
+      echo -e "${RED}Multisite not found!"
+      exit 1
+    fi
+
+    # Check SUBDOMAIN_INSTALL value
+    SUBDOMAIN_INSTALL=$(terminus wp "${site_id}".dev -- config get SUBDOMAIN_INSTALL)
+    if [ "${type}" == 'subdir' ]; then
+      # SUBDOMAIN_INSTALL should be false.
+      if [ "${SUBDOMAIN_INSTALL}" == "1" ]; then
+        echo -e "${RED}Subdirectory configuration not found!"
+        exit 1
+      fi
+    fi
+    if [ "${type}" == 'subdom' ]; then
+      # SUBDOMAIN_INSTALL should be true.
+      if [ "${SUBDOMAIN_INSTALL}" != "1" ]; then
+        echo -e "${RED}Subdomain configuration not found!"
+        exit 1
+      fi
+    fi
+  fi
+}
+
+set_up_subsite() {
+  if [ "${type}" != 'single' ]; then
+    echo ""
+    echo -e "${YELLOW}Set up subsite${RESET}"
+    # Set a URL var for the type of multisite.
+    if [ "${type}" == 'subdom' ]; then
+      URL="foo.dev-${site_id}.pantheonsite.io"
+    fi
+    if [ "${type}" == 'subdir' ]; then
+      URL="${site_url}/foo"
+    fi
+
+    # Check if the sub-site already exists.
+    EXISTING_SITE=$(terminus wp "${site_id}".dev -- site list --field=url | grep -w "foo" || true)
+
+    if [ -z "$EXISTING_SITE" ]; then
+      # Create the sub-site only if it does not already exist.
+      terminus wp "${site_id}".dev -- site create --slug=foo --title="Foo" --email="foo@dev.null"
+      terminus wp "${site_id}".dev -- option update permalink_structure '/%postname%/' --url="$URL"
+    else
+      echo -e "${YELLOW}Sub-site already exists at $URL. Skipping creation.${RESET}"
+    fi
+    terminus wp "${site_id}".dev -- option update permalink_structure '/%postname%/' --url="$URL"
+  fi
+}
```

Comment:
```suggestion
set_up_subsite() {
  if [[ "${type}" == 'single' ]]; then
    return
  fi
    echo ""
    echo -e "${YELLOW}Set up subsite${RESET}"
    # Set a URL var for the type of multisite.
    if [[ "${type}" == 'subdom' ]]; then
      URL="foo.dev-${site_id}.pantheonsite.io"
    fi
    if [[ "${type}" == 'subdir' ]]; then
      URL="${site_url}/foo"
    fi

    # Check if the sub-site already exists.
    if terminus wp "${site_id}".dev -- site list --field=url | grep -w "foo"; then
      echo -e "${YELLOW}Sub-site already exists at $URL. Skipping creation.${RESET}"
    else
      # Create the sub-site only if it does not already exist.
      terminus wp "${site_id}".dev -- site create --slug=foo --title="Foo" --email="foo@dev.null"
      terminus wp "${site_id}".dev -- option update permalink_structure '/%postname%/' --url="$URL"      
    fi
    terminus wp "${site_id}".dev -- option update permalink_structure '/%postname%/' --url="$URL"
}
```
looks like you're not actually doing anything with EXISTING_SITE?